index.html

```diff
@@ -2281,6 +2283,51 @@ <h3>Status</h3>
 }
         </pre>
 
+        <p>
+It is possible for a [=credential=] to have more than one status associated
+with it, such as whether it has been revoked or suspended.
+        </p>
+
+        <pre class="example nohighlight"
+          title="Usage of multiple entries for the status property">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://license.example/credentials/9837",
+  "type": ["VerifiableCredential", "ExampleDrivingLicenseCredential"],
+  "issuer": "https://license.example/issuers/48",
+  "validFrom": "2020-03-14T12:10:42Z",
+  "credentialSubject": {
+    "id": "did:example:f1c276e12ec21ebfeb1f712ebc6",
+    "license": {
+      "type": "ExampleDrivingLicense",
+      "name": "License to Drive a Car"
+    }
+  },
+  <span class="highlight">"credentialStatus": [{
+    "id": "https://license.example/credentials/status/84#14278",
+    "type": "BitstringStatusListEntry",
+    "statusPurpose": "revocation",
+    "statusListIndex": "14278",
+    "statusListCredential": "https://license.example/credentials/status/84"
+  }, {
+    "id": "https://license.example/credentials/status/84#82938",
+    "type": "BitstringStatusListEntry",
+    "statusPurpose": "suspension",
+    "statusListIndex": "82938",
+    "statusListCredential": "https://license.example/credentials/status/84"
+  }]</span>
+}
+        </pre>
+
+        <p>
+Implementers are cautioned that [=credentials=] with multiple status entries
+might contain conflicting information and reconciling those conflicts is a
+part of the [=validation=] process.
```

Comment:
The definition for "validation" says nothing about credential status and this sentence doesn't help anyone who gets here looking for an answer to "how do I reconcile conflicting status information". It's fine if this is out of scope for VCDM but it should probably be explicitly stated.

Reply:
@TallTed addressed your concern by adding some text in https://github.com/w3c/vc-data-model/pull/1450/commits/6ee313135e1f4efb95f5396658f039c7bbfe0c45.